src/Components/projects.json

```diff
@@ -0,0 +1,53 @@
+{
+  "tags_to_project": {
+    "homepage_carousel": [0, 1],
+    "internal_projects": [0],
+    "external_projects": [1] 
+  },
+  "proj_list": [
+    {
+      "project_name": "project 1",
```

Comment:
Take a look at `src/Components/ProjectsPage/Projects.js` and see if you can use those directly instead. Feel free to add a image component in there